operator/go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	github.com/quasilyte/go-ruleguard/dsl v0.3.22
 	github.com/redpanda-data/common-go/net v0.1.0
 	github.com/redpanda-data/common-go/rpadmin v0.1.13-0.20250109154132-12ac78a58f95
+	github.com/redpanda-data/common-go/secrets v0.1.2
```

Comment:
Please bump to 0.1.3 per 
https://github.com/redpanda-data/redpanda-operator/pull/651
https://github.com/redpanda-data/common-go/pull/52